packages/kbn-es/src/cluster.js

```diff
@@ -558,4 +562,30 @@ exports.Cluster = class Cluster {
     }
     return esJavaOpts.trim();
   }
+
+  /**
+   * Run an Elasticsearch Serverless Docker cluster
+   *
+   * @param {ServerlessOptions} options
+   */
+  async runServerless(options = {}) {
+    if (this._process || this._outcome) {
+      throw new Error('ES has already been started');
+    }
+
+    await runServerlessCluster(this._log, options);
+  }
+
+  /**
+   * Run an Elasticsearch Docker container
+   *
+   * @param {DockerOptions} options
+   */
+  async runDocker(options = {}) {
```

Comment:
Do we need to account for 
```
  reportTime(startTime, 'ready', {
    success: true,
  });
```

Reply:
The `execa` promise only resolves if we run docker detached, which loses automatic log outputs to the users. I've tried a few things, but nothing worked. I agree better logging and reporting would be good just not sure how to handle it.

Reply:
:+1: I'm okay with circling back later if there's a need for the metric.